opencog/cython/opencog/atomspace_details.pyx

```diff
@@ -411,8 +435,10 @@ cdef api object py_atomspace(cAtomSpace *c_atomspace) with gil:
     cdef AtomSpace atomspace = AtomSpace_factory(c_atomspace)
     return atomspace
 
-cdef api object py_atom(UUID uuid, object atomspace):
-    cdef Handle temphandle = Handle(uuid)
-    cdef Atom atom = Atom(temphandle, atomspace)
+cdef api object py_atom(void* atom_ptr, object atomspace):
```

Comment:
This was originally:
```
cdef api object py_atom(void* atom_ptr_ptr, object atomspace):
```
which made it clearer in this context that we are not dealing with an `AtomPtr` but an `AtomPtr*` that is being passed in as a void* because Cython doesn't seem to support type casting of pointers and it needs to be a void* just below anyway.